thriftserver/thriftserver/base/kustomization.yaml

```diff
@@ -0,0 +1,127 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - thriftserver-db-dc.yaml
+  - thriftserver-db-exporter.yaml
+  - thriftserver-db-pvc.yaml
+  - thriftserver-db-secret.yaml
+  - thriftserver-db-service.yaml
+  - thriftserver-dc.yaml
+  - thriftserver-hdfs-hive-secret.yaml
+  - thriftserver-pvc.yaml
+  - thriftserver-route.yaml
+  - thriftserver-secret.yaml
+  - thriftserver-server-conf-secret.yaml
+  - thriftserver-service.yaml
+
+commonLabels:
+  opendatahub.io/component: "true"
+  component.opendatahub.io/name: thriftserver
+  component.opendatahub.io/part-of: datacatalog
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+configMapGenerator:
+  - name: thriftserver-config
+    env: params.env
+
+vars:
+  - name: namespace
+    objref:
+      kind: Service
+      apiVersion: v1
+      name: thriftserver
+    fieldref:
+      fieldpath: metadata.namespace
+  - name: storage_class
+    objref:
+      kind: ConfigMap
+      apiVersion: v1
+      name: thriftserver-config
+    fieldref:
+      fieldpath: data.storage_class
+  - name: spark_url
+    objref:
+      kind: ConfigMap
+      apiVersion: v1
+      name: thriftserver-config
+    fieldref:
+      fieldpath: data.spark_url
+  - name: spark_max_cores
+    objref:
+      kind: ConfigMap
+      apiVersion: v1
+      name: thriftserver-config
+    fieldref:
+      fieldpath: data.spark_max_cores
+  - name: spark_executor_memory
+    objref:
+      kind: ConfigMap
+      apiVersion: v1
+      name: thriftserver-config
+    fieldref:
+      fieldpath: data.spark_executor_memory
+  - name: spark_driver_memory
+    objref:
+      kind: ConfigMap
+      apiVersion: v1
+      name: thriftserver-config
+    fieldref:
+      fieldpath: data.spark_driver_memory
+  - name: s3_endpoint
+    objref:
+      kind: ConfigMap
+      apiVersion: v1
+      name: thriftserver-config
+    fieldref:
+      fieldpath: data.s3_endpoint
+  - name: database_user
+    objref:
+      kind: Secret
+      apiVersion: v1
+      name: thriftserver-db-secret
+    fieldref:
+      fieldpath: stringData.database-user
+  - name: database_password
+    objref:
+      kind: Secret
+      apiVersion: v1
+      name: thriftserver-db-secret
+    fieldref:
+      fieldpath: stringData.database-password
+  - name: database_name
+    objref:
+      kind: Secret
+      apiVersion: v1
+      name: thriftserver-db-secret
+    fieldref:
+      fieldpath: stringData.database-name
+
+configurations:
+  - params.yaml
+
+images:
+  - name: spark-cluster-image
+    newName: quay.io/opendatahub/spark-cluster-image
+    newTag: spark24
+  - name: postgresql
+    newName: openshift/postgresql
+    newTag: "9.6"
+  - name: postgres-exporter
+    newName: docker.io/wrouesnel/postgres_exporter
```

Comment:
Any chance we can use something on quay?

Reply:
Resolved